cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -32,6 +32,10 @@ spec:
   mon:
     count: 3
     allowMultiplePerNode: false
+  # set the amount of mgrs to be started, the maximum number of mgr.count is 2
```

Comment:
We can enforce a max with schema validation similar to the mons. See [this example](https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/common.yaml#L60)